backend/Dockerfile

```diff
@@ -11,7 +11,12 @@ ENV PYTHONUNBUFFERED 1
 RUN pip install --upgrade pip
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
-
+RUN pip install djangorestframework-simplejwt
+RUN pip install drf_spectacular
+RUN pip install django-debug-toolbar
+RUN pip install psycopg2-binary
+RUN pip install psycopg
+RUN pip install pillow
```

Comment:
DRY

Just use `pip install -r requirements.txt`

Reply:
@vsmutok dockerfile does not start without this files

Reply:
@vsmutok 
![image](https://github.com/mate-academy/py-cinema-full-stack/assets/104854464/6da2b5a6-535b-4358-a54f-5ad07357d6d0)

Reply:
docker-compose up --build used command

Reply:
May be I can problem with path to requirement, I have to requirements.txt and now using in backend directory, may be in main directory must use